addOns/exim/src/main/resources/org/zaproxy/addon/exim/resources/export-max.yaml

```diff
@@ -1,4 +1,4 @@
-- type: export            # Exports data into a file
```

Comment:
Worth updating the tests, e.g.:
```diff
--- a/addOns/exim/src/test/java/org/zaproxy/addon/exim/automation/ExportJobUnitTest.java
+++ b/addOns/exim/src/test/java/org/zaproxy/addon/exim/automation/ExportJobUnitTest.java
@@ -79,12 +79,18 @@ class ExportJobUnitTest extends TestUtils {
         assertThat(job.getType(), is(equalTo("export")));
         assertThat(job.getName(), is(equalTo("export")));
         assertThat(job.getOrder(), is(equalTo(AutomationJob.Order.AFTER_ATTACK)));
-        assertThat(job.getTemplateDataMin(), is(not(equalTo(""))));
-        assertThat(job.getTemplateDataMax(), is(not(equalTo(""))));
+        assertValidTemplate(job.getTemplateDataMin());
+        assertValidTemplate(job.getTemplateDataMax());
         assertThat(job.getParamMethodObject(), is(nullValue()));
         assertThat(job.getParamMethodName(), is(nullValue()));
     }
 
+    private static void assertValidTemplate(String value) {
+        assertThat(value, is(not(equalTo(""))));
+        new Yaml().load(value);
+    }
+
```

Reply:
Should the "new Yaml" bit assert does not throw?

Reply:
Yeah, that's better (the outcome is the same).